dev/com.ibm.ws.microprofile.config.1.1_fat/fat/src/com/ibm/ws/microprofile/config/fat/suite/FATSuite.java

```diff
@@ -39,18 +29,18 @@
 @RunWith(Suite.class)
 @SuiteClasses({
                 BasicConfigTests.class, //LITE
-                ClassLoadersTest.class, //FULL
-                DefaultSourcesTest.class, //FULL
-                OrdinalsForDefaultsTest.class, //FULL
-                SimultaneousRequestsTest.class, //FULL
-                SharedLibTest.class, //FULL
-                StressTest.class, //FULL
-
-                // The following don't repeat against mpConfig > 1.4. See classes for why.
-                LibertySpecificConfigTests.class, //FULL
-                CDIBrokenInjectionTest.class, //FULL
-                ClassLoaderCacheTest.class, //FULL
-                DynamicSourcesTest.class //FULL
+//                ClassLoadersTest.class, //FULL
```

Comment:
Is this intentional or just there from testing?

Reply:
This was left in from testing by mistake.